Gruvo/Gruvo/ClientApp/src/app/footer/footer.component.css

```diff
@@ -7,5 +7,5 @@
   padding-top: 10px;
   padding-bottom: 10px;
   color: white;
-  box-shadow: 0px 0px 22px 5px rgba(0,0,0,0.5) , 0 20px 20px -20px #000
+  box-shadow: 0px 0px 10px 3px rgba(0,0,0,0.5)
```

Comment:
it could be simpler if make separate special class in common styles which apply where it needs